pkg/envinfo/url.go

```diff
@@ -55,6 +55,22 @@ func (ei *EnvInfo) GetComponentPorts() ([]string, error) {
 	return ei.getPorts("")
 }
 
+func (ei *EnvInfo) completeNoPortSpecified(url *localConfigProvider.LocalURL, portsOf string, ports []string) (err error) {
```

Comment:
Maybe just `checkValidPort` as the function doesn't complete anything

Reply:
Ok updating